boards/common/nrf52xxxdk/board.c

```diff
@@ -21,8 +21,46 @@
 #include "cpu.h"
 #include "board.h"
 
+#if defined(BOARD_NRF52DK)
+#define RESET_PIN       (21U)
```

Comment:
Why pin 21 and 18?

Reply:
This must be hardware related. nrf52832dk board reset is physically connected to P0.21.
Guess its P0.18 for nrf52840dk board.

Reply:
Ah thanks.

Reply:
Ask Nordic why they have chosen them...